libcni/api_test.go

```diff
@@ -541,13 +559,33 @@ var _ = Describe("Invoking plugins", func() {
 
 				Context("equal to 0.4.0", func() {
 					It("passes a prevResult to the plugin", func() {
-						err := ioutil.WriteFile(cacheFile, []byte(`{
+						var err error
+						netConfig, err = libcni.ConfFromBytes([]byte(`{
+							"type": "noop",
+							"name": "apitest",
+							"cniVersion": "0.4.0",
+							"capabilities": {
+								"portMappings": true,
+								"somethingElse": true,
+								"noCapability": false
+							}
```

Comment:
How does this change arise?

Reply:
Because the default CNI JSON created in BeforeEach() is for 1.0.0 (current.ImplementedSpecVersion) and here we need to use a 0.4.0 JSON.